lib/assets/javascripts/utils/timeagoFactory.js

```diff
@@ -0,0 +1,17 @@
+import enGB from '../locale/en_GB/timeago';
+
+/* global timeago */
+const TimeagoFactory = (() => {
+  timeago.register('en_GB', enGB);
```

Comment:
I'm going to assume we have a plan to tie this into the user's language selection later and that we'll have ones for es, de, fr, and en_US.

Reply:
yep, we will need to add more languages through register due to the fact that we wanted to change the seconds message...